ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1378,8 +1378,18 @@ vlan_interfaces:
         source_interface: < source_interface_name >
     ipv6_enable: < true | false >
     ipv6_address: < IPv6_address/Mask >
+    # If both "ipv6_address_virtual" and "ipv6_address_virtuals" are set, all addresses will be configured
```

Comment:
Would it make sense to give the same deprecation warning as for `ipv6_virtual_router_address` for `ipv6_address_virtual` for AVD v4.0?

Reply:
yes, it would be could to be consistent!

Reply:
I have added deprecation notice to both roles for the old singular `ipv6_address_virtual` in favor of `ipv6_address_virtuals`.